import rospy

def review_drink(jobs,applications):
    combination = {}
    combination['kobuki'] = 'OrderReceiver'
    return combination 

def review_cleaning(jobs,applications):
    combination = {}
    combination['kobuki'] = 'Cleaner'
    return combination

def review_sensor(jobs,applications):
    combination = {}
    return combination
